server/spinwick.go

```diff
@@ -365,6 +571,52 @@ func (s *Server) handleDestroySpinWick(pr *model.PullRequest) {
 	}
 }
 
+func (s *Server) destroyKubeSpinWick(pr *model.PullRequest) *spinwick.Request {
+	mlog.Info("Received request to destroy kubernetes namespace")
+	request := &spinwick.Request{
+		InstallationID: "n/a",
+		Error:          nil,
+		ReportError:    false,
+		Aborted:        false,
+	}
+
+	logger := log.WithField("PR", pr.RepoName+"/!"+string(pr.Number))
```

Comment:
why we have `"/!"`

Reply:
Woops, `!<number>` is how you reference an MR/PR in Gitlab. I changed it to be format

`customer-web-server: #<prnumber>`